app/views/users/show.html.erb

```diff
@@ -1,5 +1,8 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <% if @user != current_user %>
+    <%= request_btn(@user) %>
```

Comment:
- [ ] No logic in views, let's wrap this inside another method in `users_helper.rb`

Reply:
Done